seamless-immutable/seamless.d.ts

```diff
@@ -0,0 +1,83 @@
+// Type definitions for Seamless-immutable 6.1.3
+// Project: https://github.com/rtfeldman/seamless-immutable
+// Definitions by: alex3165 <https://github.com/alex3165>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Copyright (c) 2016, Richard Feldman
```

Comment:
So we will not be able to accept this PR as long as it has this specialized licence text.